### PR TITLE
fix(accordion,stepper): adjust panel title width to align panel descriptions

### PR DESCRIPTION
**Pull request by kevinbuhmann**
CDE-2220

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [N/A] Docs have been added / updated (for bug fixes / features)
- [x] If applicable, have a visual design approval

## PR Type

Bugfix

## What is the current behavior?

Accordion and stepper panel description text was not aligned vertically.

Issue Number: CDE-2220

## What is the new behavior?

Accordion and stepper panel description text is aligned vertically.

## Does this PR introduce a breaking change?

No.

### CODE REVIEW

File: projects/angular/src/accordion/_accordion.clarity.scss [hunk 1]
```diff
@@ -114,7 +114,7 @@
   .clr-accordion-header-has-description {
     .clr-accordion-title {
       min-width: accordion-variables.$clr-accordion-step-title-min-width;
-      max-width: mixins.baselinePx(264);
+      max-width: accordion-variables.$clr-accordion-step-title-min-width;
```
Comment (valentin-mladenov) [inline projects/angular/src/accordion/_accordion.clarity.scss:117]: I found issue with this hardcoded `min` and `max` width aproach. Check [here](https://1524--storybook-clarity-design.netlify.app/?path=/story/stepper-stepper--stepper&globals=viewport:small).

Description overflows.

image

The better solution is to use percetnage based width.
Token for max width should be created as well.
Min width should not be changed.

```suggestion
      max-width: mixins.baselinePx(264);
      width: 100%;
```

**Review by valentin-mladenov — CHANGES_REQUESTED** [review]
Check my comment.

File: projects/angular/src/accordion/_accordion.clarity.scss (hunk 1 above)
Comment (kevinbuhmann) [reply projects/angular/src/accordion/_accordion.clarity.scss]: Thanks! This is fixed now.

**Review by valentin-mladenov — APPROVED** [review]
LGTM

**Pull request merged**